skan/csr.py

```diff
@@ -435,10 +433,40 @@ def path_means(self):
         means : array of float
             The average pixel value along each path in the skeleton.
         """
+
         sums = np.add.reduceat(self.paths.data, self.paths.indptr[:-1])
         lengths = np.diff(self.paths.indptr)
         return sums / lengths
 
+    def path_maximum_length(self):
+        """Compute the longest path.
+
+        Returns
+        -------
+        max_length : array of float
+            The maximum length in the skeleton.
+        """
+
+        sums = np.maximum.reduceat(self.paths.data, self.paths.indptr[:-1])
```

Comment:
```suggestion
        maxes = np.maximum.reduceat(self.paths.data, self.paths.indptr[:-1])
```
@snowformatics I think @jni means the variable name (you are computing all the max's here).